static/app/views/monitors/components/quickStartEntries.tsx

```diff
@@ -0,0 +1,94 @@
+import {Fragment} from 'react';
+import merge from 'lodash/merge';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
+
+export interface QuickStartProps {
+  dsnKey?: string;
+  orgSlug?: string;
+  slug?: string;
+}
+
+const VALUE_DEFAULTS = {
+  dsnKey: '<my-dsn-key>',
+  orgSlug: '<my-organization-slug>',
+  slug: '<my-monitor-slug>',
+};
+
+function withDefaultProps(props: QuickStartProps): Required<QuickStartProps> {
+  return merge(VALUE_DEFAULTS, props);
+}
+
+export function PythonCronQuickStart(props: QuickStartProps) {
+  const {slug} = withDefaultProps(props);
+
+  const code = `from sentry_sdk.crons import monitor
+
+# Add this decorator to instrument your python function
+@monitor(monitor_slug='${slug}')
+def tell_the_world(msg):
+  print(msg)`;
+
+  return <CodeSnippet language="python">{code}</CodeSnippet>;
+}
+
+export function CLICronQuickStart(props: QuickStartProps) {
+  const {slug, dsnKey} = withDefaultProps(props);
+
+  const script = `# Example for a Python script:
+export SENTRY_DSN=${dsnKey}
+sentry-cli monitors run ${slug} -- python path/to/file`;
```

Comment:
```suggestion
  const script = `SENTRY_DSN=${dsnKey} sentry-cli monitors run ${slug} -- python path/to/file`;
```

wdyt we simplify and keep it all under in one line?

Reply:
It will be very long then